tests/caasp/stack_update.pm

```diff
@@ -67,7 +67,8 @@ sub run {
     die "Admin should be updated already" if check_screen 'velum-update-admin', 0;
     assert_and_click "velum-update-all";
 
-    assert_screen 'velum-bootstrap-done', $nodes * 150;
+    # 5 minutes per node
+    assert_screen 'velum-bootstrap-done', $nodes * 300;
```

Comment:
Can you provide update test that fails on this timeout?
If yes it's ok to merge. If not please don't change this because we will start hitting 2h timeout on bigger clusters if we are not careful.

Reply:
I gave it to you to IRC, but for courtesy reasons I will put it here also: http://skyrim.qam.suse.de/tests/83#step/stack_update/15